package/gluon-client-bridge/luasrc/lib/gluon/upgrade/320-gluon-client-bridge-wireless

```diff
@@ -112,4 +120,16 @@ wireless.foreach_radio(uci, function(radio, index, config)
 	configure_owe_transition_mode(config, radio_name)
 end)
 
+if not sysconfig.gluon_version and has_client_radio then
```

Comment:
Checks for `sysconfig.gluon_version` are almost always wrong. It's better to delete and recreate as much config as possible on each reconfigure instead of trying to handle the first run differently.

All new config that needs to persist should be stored in `/etc/config/gluon` - eventually I'd like to make that the only file that is preserved on upgrades at all.

Reply:
I'd add an optional 3rd argument to `is_disabled()` to extend the logic in the case where the UCI section does not exist yet.